src/components/Table.jsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+import { DataGrid } from "@mui/x-data-grid";
+import style from "./Table.module.css";
+class DataGridDemo extends React.Component {
+  render() {
+    return (
+      <div style={{ height: 300, width: "90%" }}>
```

Comment:
Try to move inline css to separate file where your styles are stored and use a class for that